client/src/components/fetchCustomers.js

```diff
@@ -0,0 +1,13 @@
+import { gql } from "@apollo/client";
+
+const FETCH_CUSTOMERS = gql`
```

Comment:
What's the rationale behind putting this in a separate file? Keeping GQL queries in one place make them easier to find unless you have a good reason to keep this one separate.

Reply:
I have used this query for two times in different components, so I put it in a seperate file.  Is it a good reason to do it?